_docs/style/text-conventions/date-time.md

```diff
@@ -0,0 +1,215 @@
+
+{% endcolumn %}
+
+</div>
+
+</section>
+
+<section class="static-section"  markdown="1">
+
+<div class="hxRow" markdown="1">
+
+{% column left:"hxCol hxSpan-12-xs hxSpan-12-sm hxSpan-12-md hxSpan-12-lg" %}
+
+## Time
+
+You can show time by using either the 24-hour or 12-hour clock.
+
+- If the Control Panel that you're documenting shows or uses the 12-hour clock, then be consistent with the interface and use the 12-hour clock.
+- The 24-hour clock is the preferred format for international audiences and the format used in most computer systems, so use the 24-hour clock when possible.
+
+{% endcolumn %}
+
+</div>
+
+</section>
+
+<section class="static-section"  markdown="1">
+
+<div class="hxRow" markdown="1">
+
+{% column left:"hxCol hxSpan-12-xs hxSpan-12-sm hxSpan-8-md hxSpan-8-lg" %}
+
+### 12-hour clock
+
+When you use the 12-hour clock to show time, use the following guidelines:
+
+- Use a colon to separate the hours and minutes. If the minutes are 00, you don't need to show them unless you're showing a span of time that includes a time with minutes.
+- Use lowercase letters for abbreviations of ante meridiem (a.m.) and post meridiem (p.m.). Separate these abbreviations from the time with a space. Use periods in the abbreviations.
+- When specifying time zones, show both the spelled-out name and the abbreviation. Show the name in lowercase letters; use uppercase letters and no periods for the abbreviation.
+- Avoid references to standard and daylight saving time because the appropriate designation changes frequently. However, if you need to include such a reference, insert *S* (for standard) or *D* (for daylight) as the second character in the abbreviation.
+- When referring to 12 a.m., use **12 midnight** or just **midnight**. When referring to 12 p.m., use **12 noon** or just **noon**.
+
+Consider the following examples for 12-hour clock time:
+
+- 10:29 a.m.
+- 6 p.m.
+- 6:00 p.m. to 8:30 p.m.
+- 10:30 a.m. central time (CT)
+- 1:30 p.m. central standard time (CST)
+- midnight
+
+{% endcolumn %}
+
+</div>
+
+</section>
+
+<section class="static-section"  markdown="1">
+
+<div class="hxRow" markdown="1">
+
+{% column left:"hxCol hxSpan-12-xs hxSpan-12-sm hxSpan-8-md hxSpan-8-lg" %}
+
+### 24-hour clock
+
+When you use the 24-hour clock to show time, use the following guidelines:
+
+- Use a colon to separate the hours, minutes, and seconds.
+- Show the hours, minutes, and seconds with two digits each, even if the leading digit is 0.
+- When possible, display the local time as specified in user preferences. If time preferences have not been set, use the assumed time based on the browser. Only use UTC when no timezone preference can be determined or standardization across timezones is necessary.
```

Comment:
@palindromebart - this one point is a holdover from the original date/time content. I'm not sure if it's true or helpful. I think you were investigating with one of the dev team about this? Let's discuss...